index.js

```diff
@@ -10,6 +10,13 @@ const Walker = require('node-source-walk');
  * @return {String[]}
  */
 module.exports = function(src, options = {}) {
+
+  // Determine whether to skip "type-only" imports
+  // https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html#import-types
+  const skipTypeImports = Boolean(options.skipTypeImports);
+  // Remove skipTypeImports option, as this option may not be recognized by the walker/parser
+  delete options.skipTypeImports;
```

Comment:
We should not mutate the options object. Make a copy of the object before removing the property. Like `const walkerOptions = Object.assign({}, options);`

Reply:
Thanks. Addressed in 259601c9120c5137fb6c9dd76dedaca0e7282c08.